_stbt/gst_utils.py

```diff
@@ -168,8 +168,27 @@ def __del__(self):
             self._ctx.__exit__(None, None, None)
 
 
+class CapturedFrame(numpy.ndarray):
```

Comment:
Presumably we're going to make this a public API at some future point (#364 perhaps).